src/Npgsql/TypeMapping/GlobalTypeMapper.cs

```diff
@@ -634,6 +634,17 @@ void SetupLTreeHandlers()
                 }.Build());
             }
 
+            void SetupCubeHandlers()
```

Comment:
nit: we can probably just move this into SetupMiscHandlers as there's only one mapping.